metals/src/main/scala/scala/meta/internal/metals/debug/SetBreakpointsRequestHandler.scala

```diff
@@ -109,14 +108,9 @@ private[debug] final class SetBreakpointsRequestHandler(
     val source = DebugProtocol.copy(request.getSource)
     source.setPath(s"dap-fqcn:$fqcn")
 
-    val lines = breakpoints
-      .map(_.getLine: Int)
-      .distinct
-
     val partition = new SetBreakpointsArguments
     partition.setBreakpoints(breakpoints)
     partition.setSource(source)
-    partition.setLines(lines)
```

Comment:
I want to do some testing locally on these, but reading through it, it makes it seem that the lines should always be accessed through the breakpoints themselves, and that's why the `getLines` is also deprecated. So I don't believe this is even needed. I need to test a bit more locally to make sure this is still all working as expected.

Reply:
Testing locally with `nvim-metals` this all seems to work fine, but can someone with VS Code test as well?

Reply:
Looks to work ok! I think it was just added in case it was needed indeed.